template-for-repository/proofs/Makefile.common

```diff
@@ -119,7 +119,7 @@ PROOFDIR ?= $(abspath .)
 CBMCDIR ?= $(PROOF_ROOT)/cbmc
 
 # Default CBMC flags used for property checking and coverage checking
-CBMCFLAGS += --unwind 1 $(CBMC_UNWINDSET) --flush
+CBMCFLAGS += --unwind 1 --flush
```

Comment:
We should add ac comment explaining why we need `--unwind 1` here.

Reply:
Good idea; updated.